frontend/model/state.js

```diff
@@ -21,6 +19,7 @@ import chatroomModule from '~/frontend/model/chatroom/vuexModule.js'
 
 // Wrapper function for performing contract upgrades and migrations
 // TODO: Consider moving this function into a different file
+// eslint-disable-next-line no-unused-vars
```

Comment:
Could you add to the comment here a link to this PR that says something like, "to see examples how to use this function, see this PR"?